website/server/controllers/api-v3/challenges.js

```diff
@@ -410,13 +541,17 @@ api.exportChallengeCsv = {
 };
 
 /**
- * @api {put} /api/v3/challenges/:challengeId Update a challenge
+ * @api {put} /api/v3/challenges/:challengeId Update a challenge. Only reflects
+ * changes to the full name of the challenge. Other challenge details cannot be updated after creation.
+ *
  * @apiName UpdateChallenge
  * @apiGroup Challenge
  *
  * @apiParam {UUID} challengeId The challenge _id
+ * @apiParam {String} [challenge.name] The full name of the challenge.
  *
  * @apiSuccess {Object} data The updated challenge
+ * @apiUse ChallengeSuccessExample
  *
  * @apiUse ChallengeNotFound
```

Comment:
@crookedneighbor Added the `@apiPermission`. Also added a `MustBeChallengeLeader` error on line 560. LMK if I should leave that in, or if it's redundant.